rib/reconciler/reconcile.go

```diff
@@ -339,13 +345,13 @@ func diff(src, dst *rib.RIB, explicitReplace map[spb.AFTType]bool) (*reconcileOp
 // v4Operation builds a gRIBI IPv4 operation with the specified method corresponding to the
 // prefix pfx in network instance ni, using the specified ID for the operation. The contents
 // of the operation are the entry e.
-func v4Operation(method spb.AFTOperation_Operation, ni, pfx string, id uint64, e *aft.Afts_Ipv4Entry) (*spb.AFTOperation, error) {
+func v4Operation(method spb.AFTOperation_Operation, ni, pfx string, id *atomic.Uint64, e *aft.Afts_Ipv4Entry) (*spb.AFTOperation, error) {
```

Comment:
pfx doesn't seem to be used to obtain the actual prefix but rather the AFT entry is, is pfx only used for an possible error message?

Reply:
That's correct -- I made it such that we just read from the proto here, and in the other `XXOperation` calls.